Tests/test_excinfo.py

```diff
@@ -371,94 +370,27 @@ def f():
         
         g = f()
         self.A(None)
-        self.assertEqual(g.next(), 1)
+        self.assertEqual(next(g), 1)
         self.A(None)
         try:
             try:
                 g.throw(ValueError(87))
-                Assert(False)
+                assert False
             finally:
                 # exceptional path.
-                # exc_info should have been cleared on exiting generator.
-                self.A(None)
+                # The generator throws an exception, causing us
+                # to receive it
+                self.A(22)
         except:
             self.A(22)
-        self.A(22)
-
-
-    #---------------------------------------------------------------------
-    #
-    # Test sys.exc_clear(), which was added in Python 2.3
-    # This clears the last exception status.
-    #
-    #---------------------------------------------------------------------
-
-    def test_clear_simple(self):
-        """simple case of clear in an except block."""
-        try:
-            raise ValueError(12)
-        except:
-            self.A(12)
-            sys.exc_clear()
-            self.A(None)
-        self.A(None)
-
-    def test_clear_nested(self):
-        try:
-            raise ValueError(13)
-        except:
-            try:
-                self.A(13)
-                raise ValueError(54)
-            except:
-                self.A(54)
-                sys.exc_clear()
-                self.A(None)
-            self.A(None)
-        self.A(None)
-
-    def test_clear_nested_func(self):
-        def f():
-            try:
-                self.A(13)
-                raise ValueError(54)
-            except:
-                self.A(54)
-                sys.exc_clear()
-                self.A(None)
-            self.A(None)  # will be restored after func returns
-        #
-        try:
-            raise ValueError(13)
-        except:
-            self.A(13)
-            f()  # calls sys.exc_clear()
-            self.A(13)  # still restored even after clear
-        self.A(13)
-
-
-    # Test clearing when there isn't an active exception (outside except block)
-    def test_clear_no_active_ex(self):
-        self.A(None)
-        sys.exc_clear()
-        self.A(None)
-        try:
-            sys.exc_clear()
-            self.A(None)
-        except:
-            pass
-        try:
-            pass
-        finally:
-            sys.exc_clear()
-            self.A(None)
         self.A(None)
-
+   
     #========================================================
     # With's Pep (http://www.python.org/dev/peps/pep-0343/) says the
     # __exit__ can be invoked by an except block,
     # but unlike a normal except, that shouldn't set sys.exc_info().
 
+    @unittest.skip('unbound variable: $localContext')
```

Comment:
Presumably this is the same issue as https://github.com/IronLanguages/ironpython3/issues/451? Maybe add a link for searchability.